src/app/Instance/BridgeOverview/Components/BODashboardTableView.tsx

```diff
@@ -128,7 +131,23 @@ export const BODashboardTableView = (
                 </Stack>
               </Td>
               <Td isActionCell>
-                {rowActions ? <ActionsColumn items={rowActions} /> : null}
+                {
+                  <ActionsColumn
+                    items={[
+                      {
+                        title: "Edit",
+                        isDisabled: !canEditResource(item.status),
+                        onClick: () => onEditProcessor(item.id),
+                      },
+                      {
+                        title: "Delete",
+                        isDisabled: !item || !canDeleteResource(item.status),
+                        onClick: (): void =>
+                          deleteProcessor(item.id, item.name),
+                      },
```

Comment:
The `void` return type here is unnecessary, it will be inferred by TS. As it happens in the other action where it is not set.